packages/payment-integration/src/index.ts

```diff
@@ -1,17 +1,23 @@
+export { AddressRequestBody } from './address';
 export { BillingAddress, BillingAddressRequestBody } from './billing';
 export { Cart } from './cart';
 export { Checkout } from './checkout';
 export { StoreConfig } from './config';
-export { Customer } from './customer';
+export { Customer, CustomerRequestOptions, CustomerInitializeOptions, ExecutePaymentMethodCheckoutOptions } from './customer';
 export { InvalidArgumentError,
+    MissingDataError,
+    MissingDataErrorType,
+    NotImplementedError,
     NotInitializedError,
     NotInitializedErrorType,
     OrderFinalizationNotRequiredError,
     PaymentArgumentInvalidError,
     PaymentMethodCancelledError } from './errors';
 export { Order, OrderRequestBody } from './order';
-export { CardInstrument, Payment, PaymentInitializeOptions, PaymentMethod, PaymentRequestOptions, PaymentStrategyNew } from './payment';
+export { CardInstrument, Payment, PaymentInitializeOptions, PaymentMethod, PaymentRequestOptions } from './payment';
 export { default as PaymentIntegrationSelectors } from './payment-integration-selectors';
 export { default as PaymentIntegrationService } from './payment-integration-service';
-export { Consignment, ShippingAddress, ShippingAddressRequestBody } from './shipping';
-export { PaymentIntegrationServiceMock, getCart, getCheckout, getConfig, getOrderRequestBody } from './test-utils';
+export { Consignment, ShippingAddress, ShippingOption, ShippingAddressRequestBody } from './shipping';
+export { CustomerWalletButtonStrategy, PaymentStrategyNew } from './strategy';
+export { PaymentIntegrationServiceMock, getCart, getCheckout, getConfig, getConsignment, getOrderRequestBody, getShippingOption } from './test-utils';
```

Comment:
I just noticed this. I think mock files should be exported via different file. Not sure how smart Webpack is but ideally non-test code shouldn't be referring code written for tests, even though the reference is indirect.

Reply:
I am not sure if that is even possible, the only way I feel this can be possible is to move the mocks to a separate package?